### DISCUSSION TITLE
Change Book name

### DISCUSSION THREAD

**Discussion by clinklet** [ezra-bible-app/ezra-bible-app#1159] [Q&A] [answered]
Is there any way to change the name of a book?  Revelation of John is just really bad.  The book is about the Revelation of Jesus the Christ (as stated in verse 1).  I was able to change the display from John to Jesus Christ (in book_selection_menu.html file), but when I do a copy/paste into documents, it still has the "Revelation of John" when I do the paste.

Thanks everyone!

**Comment by tobias-klein** [comment, marked as answer]
@clinklet 
The SWORD library is used to get book names for the respective language. The function used in Ezra is coming from `node-sword-interface` and is called `getSwordTranslation`, see https://github.com/ezra-bible-app/ezra-bible-app/blob/master/app/backend/ipc/ipc_nsi_handler.js#L339.
This function is also used to localize the book selection menu.
See https://github.com/ezra-bible-app/ezra-bible-app/blob/master/app/frontend/components/book_selection_menu.js#L124
